### ISSUE TITLE
 malformed attribute in conversion summary

### ISSUE THREAD

**Issue by davidcarlisle** [arXiv/html_feedback#281]
### Description

The file has one spurious "  on line 48 of https://browse.arxiv.org/html/2401.05361v1
```
found"=""
```

This  is in the added box documenting conversions so not really part of the conversion itself but does prevent easy processing/checking of the file as it's a fatal error for xml tools

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

9ccf5035-ed1a-449c-936f-84f743f2c0e7

### Paper URL

https://browse.arxiv.org/html/2401.05361v1

### Browser

Firefox/121.0

### Device Type

Desktop

**Comment by dginev** [comment]
The line in question is:
```xml
<div aria-label="”Conversion" been="" class="package-alerts ltx_document" errors="" found”="" have="" role="“status”">
```

Likely a typo in the page header, I'm just noting that it's not latexml related.

It could be helpful to do a pass with nuvalidator to double-check for any other markup wrinkles in this doc:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fbrowse.arxiv.org%2Fhtml%2F2401.05361v1

**Comment by davidcarlisle** [comment]
> It could be helpful to do a pass with nuvalidator to double-check for any other markup wrinkles in this doc: https://validator.w3.org/nu/?doc=https%3A%2F%2Fbrowse.arxiv.org%2Fhtml%2F2401.05361v1

all good apart from invalid attributes `have`, `been` and `found ` all on that same line, this is presumably intended to have been a text string `have been found` that's got scrambled into attributes somewhere along the line?

**Comment by cbf66** [comment]
Yup. We took some text straight from doc person not realizing it had “smart quotes” :-). 

https://github.com/arXiv/arxiv-view-as-html/blame/d312f8e9b87f0ca598dfa449ee7394039fe5ebfc/ConversionContainer/source/convert/__init__.py#L281

@mnazzaro will fix ASAP.

**Comment by davidcarlisle** [comment]
@cbf66  incidentally you may be interested in what I was processing this document for:

https://github.com/latex3/tagging-project/discussions/56#discussioncomment-8132024

**Comment by dginev** [comment]
This example is resolved on regenerating the article. The header will get revised near completely soon, closing here.
